storage/Devices/BcacheImpl.cc

```diff
@@ -152,6 +240,15 @@ namespace storage
 	const File size_file = prober.get_system_info().getFile(SYSFS_DIR + get_sysfs_path() + "/size");
 
 	set_region(Region(0, size_file.get<unsigned long long>(), 512));
+
+	const File cache_mode_file = prober.get_system_info().getFile(SYSFS_DIR + get_sysfs_path() + "/bcache/cache_mode");
+	set_cache_mode(parse_mode(cache_mode_file));
+
+	const File writeback_percent_file = prober.get_system_info().getFile(SYSFS_DIR + get_sysfs_path() + "/bcache/writeback_percent");
+	set_writeback_percent(writeback_percent_file.get<unsigned>());
+
+	const File sequential_cutoff_file = prober.get_system_info().getFile(SYSFS_DIR + get_sysfs_path() + "/bcache/sequential_cutoff");
+	set_sequential_cutoff(parse_size(sequential_cutoff_file));
```

Comment:
When ```prober.get_system_info()``` is used several times it can be saved in a reference: ```SystemInfo& system_info = prober.get_system_info();```

Reply:
ok, will modify